src/ubuntu/16.04/cross/arm64-alpine/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-crossdeps
+
+# Install binutils-aarch64-linux-gnu
+#
+# Also install bison. This is required to build musl-cross-make
+RUN apt-get update \
+    && apt-get install -y \
+        binutils-aarch64-linux-gnu \
+        bison \
+    && rm -rf /var/lib/apt/lists/*
+
+# Workaround to build a functioning ld.gold which can link for linux-musl arm64
+RUN cd /tmp && git clone https://github.com/richfelker/musl-cross-make.git
```

Comment:
Hmm, multiline suggestions don't work. 
Here is what you can use instead. Please note that it is preferred practice to use a single RUN command instead of multiple ones as each RUN creates a layer in the resulting image.
```
# Workaround to build a functioning ld.gold which can link for linux-musl arm64
RUN cd /tmp \
    && wget https://ftp.gnu.org/gnu/binutils/binutils-2.27.tar.gz \
    && tar -xf binutils-2.27.tar.gz \
    && cd binutils-2.27 \
    && ./configure --disable-werror --target=aarch64-alpine-linux-musl --prefix=/usr --libdir=/lib --disable-multilib --with-sysroot=aarch64-alpine-linux-musl --enable-gold=yes --program-prefix=aarch64-alpine-linux-musl- \
    && make \
    && make install \
    && cd .. \
    && rm -r *
```

Reply:
HI Jan, I had to rebuild several times and there were issues during the build. I kept the run commands separate to allow for restore points; however, thinking further I do not believe many people, if any build the images out of this repo manually.

Reply:
That's the way I also usually do stuff in Dockerfile. I just convert it to the minimal number of RUN commands when it is ready for PR. @MichaelSimons advised me on keeping a minimal muber of RUN commands a long time ago.

Reply:
Makes sense thanks!

Reply:
As Jan pointed out it is general best practice to reduce the number of layers.  Because of the way docker utilizes a union file system and layers, you must do this in scenarios where you want to cleanup temporary artifacts.  If not, you end up caring the deleted files in a base layer anyways which defeats the purpose.  Docker does provide the ability to consolidate all image layers into one which is useful in some scenarios.  I agree though that often times during development iterations, it is beneficial to break the layers apart.